cli/search/search.go

```diff
@@ -0,0 +1,67 @@
+package search
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/apex/log"
+)
+
+// isDeprecated checks if the program version is lower than 1.9.0.
+func isDeprecated(version string) bool {
+	splitedVersion := strings.Split(version, ".")
+	if len(splitedVersion) < 2 {
+		return false
+	}
+	if splitedVersion[0] == "1" && len(splitedVersion[1]) < 2 {
+		return true
+	}
+	return false
+}
+
+// SearchVersions outputs available versions of program.
+func SearchVersions(program string) error {
+	var cmd *exec.Cmd
```

Comment:
I think we should check if `git' is installed on the host.
In the case of production `git' may be absent.

Reply:
Example from current PR
https://github.com/tarantool/tt/pull/119/files#diff-0d5489a394b59a95c5e89f46fbf9afb6715c790a6ddb80626e99a917c508dedcR43

Reply:
Added